Added City of Ashes (issue #21)

MikeLenoir opened the pull request:
> Please check all tasks that you've completed by placing an `x` within the square braces below.

## Proposed Changes
- [x] I've read the [CODE OF CONDUCT](https://github.com/my-first-pr/hacktoberfest-2018/blob/master/CODE_OF_CONDUCT.md) and abide to it.
- [ x] I've read the [CONTRIBUTING.md](https://github.com/my-first-pr/hacktoberfest-2018/blob/master/CONTRIBUTING.md)
- [x] I've forked the repository.
- [x] I've created a branch and made my changes in it.
- [x] I understand opening a PULL REQUEST doesn't mean it will be merged for sure.

MikeLenoir committed:
Added Lord of The Flies

MikeLenoir committed:
Added City of Ashes

niftyrinki commented on city-of-ashes.json:
```diff
@@ -0,0 +1,12 @@
+{
+  "title": "City of Ashes (The Mortal Instruments Series #2)",
```
Please do not add series name. Only the original book title is good enough.

niftyrinki reviewed (changes requested):
Please mention the issues which are fixed by this PR.

MikeLenoir committed:
Removed series name from City of Ashes

MikeLenoir commented:
I updated the title and pushed the changes, does it automatically push to this pull request? Added the issue to the title, not sure of the proper way of indicating this.

niftyrinki committed:
Delete lord-of-the-flies.json
